back/engines/commercial/report_builder/spec/acceptance/reports_spec.rb

```diff
@@ -216,6 +218,14 @@
         end
       end
 
+      describe 'updating the visibility of a report' do
+        example 'Visibility successfully updates by report id' do
+          do_request(report: { visible: false })
```

Comment:
```suggestion
        example 'Visibility successfully updates by report id' do
          expect(report.reload.visible).to be(true)
          do_request(report: { visible: false })
```
I usually add such check to make sure the test actually changes this value. But not sure it's very conventional, so feel free to reject this change.